Documentation/using-rkt-with-systemd.md

```diff
@@ -62,3 +64,131 @@ Environment=TMPDIR=/var/tmp
 ExecStart=/usr/bin/rkt run --inherit-env --private-net --port=http:8888 myapp.com/myapp-1.3.4
 Restart=always
 ```
+
+## Using (not only) systemd tools
+
+Let's assume that from now on, the service from the simple example unit file is started on the host.
+
+### machinectl list
+
+By using systemd-nspawn, we have integration with systemd-machined for free. Note the machine name - it will show up in snippets later too.
+
+```
+MACHINE                          CONTAINER SERVICE
+rkt-6d0d9608-a744-4333-be21-942145a97a5a container nspawn
+
+1 machines listed.
+```
+
+## ps auxf
+
+The snippet below taken from output of `ps auxf` shows that rkt executes stage1's systemd-nspawn.
+
+```
+USER       PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
+root      7258  0.2  0.0  19680  2664 ?        Ss   12:38   0:02 stage1/rootfs/usr/lib/ld-linux-x86-64.so.2 stage1/rootfs/usr/bin/systemd-nspawn --boot --register=true --link-journal=try-guest --quiet --keep-unit --uuid=6d0d9608-a744-4333-be21-942145a97a5a --machine=rkt-6d0d9608-a744-4333-be21-942145a97a5a --directory=stage1/rootfs -- --default-standard-output=tty --log-target=null --log-level=warning --show-status=0
+root      7275  0.0  0.0  27348  4316 ?        Ss   12:38   0:00  \_ /usr/lib/systemd/systemd --default-standard-output=tty --log-target=null --log-level=warning --show-status=0
+root      7277  0.0  0.0  23832  6100 ?        Ss   12:38   0:00      \_ /usr/lib/systemd/systemd-journald
+root      7343  0.3  0.0  10652  7332 ?        Ssl  12:38   0:04      \_ /etcd
+```
+
```

Comment:
Maybe explain a bit what we see there, like:

Here we see systemd-nspawn with a systemd child, we don't see rkt because we exec nspawn instead of forking, its child is...